### PR TITLE
Features + Bugfixes

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/controllers/de/laser/LoginController.groovy [hunk 1]
```diff
@@ -186,14 +186,32 @@ class LoginController {
...
-          mailSendService.sendMailToUser(user, 'Passwortänderung', '/mailTemplates/text/newPassword', [user: user, newPass: newPassword])
+          mailSendService.sendMailToUser(user, message(code: 'email.subject.forgottenPassword'), '/mailTemplates/text/newPassword', [user: user, newPass: newPassword])
         }
       }
       else flash.error = g.message(code:'menu.user.forgottenPassword.userError') as String
     }
     redirect action: 'auth'
   }
 
+    @Transactional
```
Comment (croqueGrec09) [inline grails-app/controllers/de/laser/LoginController.groovy:197]: Trnsactional?

**Review by croqueGrec09 — COMMENTED** [review]
one remark

File: grails-app/controllers/de/laser/LoginController.groovy (hunk 1 above)
Comment (Moetez16) [reply grails-app/controllers/de/laser/LoginController.groovy:197]: look at 

@Transactional
def resetForgottenPassword() {

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**